src/python/pants/notes/2.13.x.md

```diff
@@ -1,5 +1,85 @@
 # 2.13.x Release Series
 
+## 2.13.0.dev5 (Jun 07, 2022)
+
+### New Features
+
+* Add `extra_env_vars` field to `experimental_shell_command` target. ([#15742](https://github.com/pantsbuild/pants/pull/15742))
+
+* Add `experimental-explorer` builtin goal. ([#15695](https://github.com/pantsbuild/pants/pull/15695))
+
+### User API Changes
+
+* Always ignore paths which are not files, directories, or links. ([#15773](https://github.com/pantsbuild/pants/pull/15773))
+
+* Add API types and rules to cli help info ([#14227](https://github.com/pantsbuild/pants/pull/14227))
+
+* Change the default `remote_cache_warnings` warnings behavior to `backoff`. ([#15745](https://github.com/pantsbuild/pants/pull/15745))
+
+* Revert "Go can use both `file` and `resource` targets (#15663)" ([#15725](https://github.com/pantsbuild/pants/pull/15725))
+
+* Deprecate `--exclude-target-regexp` in favor of `--filter-address-regex` ([#15729](https://github.com/pantsbuild/pants/pull/15729))
+
+### Plugin API Changes
+
+* Require setting `description_of_origin` with `AddressInput` and `UnparsedAddressInputs` ([#15743](https://github.com/pantsbuild/pants/pull/15743))
+
+* Require keyword arguments for `AddressInput` ([#15741](https://github.com/pantsbuild/pants/pull/15741))
+
+### Bug fixes
+
+* `tailor` should skip bad requirements targets ([#15755](https://github.com/pantsbuild/pants/pull/15755))
+
+* `tailor` for Go does not add targets when no ancestor `go.mod` ([#15750](https://github.com/pantsbuild/pants/pull/15750))
+
+* Record metrics for remote cache store errors ([#15744](https://github.com/pantsbuild/pants/pull/15744))
+
+* Fix help for `--process-total-child-memory-usage` and `--process-per-child-memory-usage` ([#15733](https://github.com/pantsbuild/pants/pull/15733))
+
+* Fix `--changed-since` with directories unknown to Pants ([#15714](https://github.com/pantsbuild/pants/pull/15714))
+
+### Documentation
+
+* Improve error messages for invalid addresses ([#15751](https://github.com/pantsbuild/pants/pull/15751))
+
+* Correctly describe the origin of invalid specs, e.g. `--paths-from` ([#15730](https://github.com/pantsbuild/pants/pull/15730))
+
+### Internal (put these in a PR comment for review, not the release notes)
```

Comment:
I see a whole lot of internal changes here... :)

Reply:
😅